dts/arm/silabs/xg28/xg28.dtsi

```diff
@@ -195,6 +197,19 @@
 			pstate_em2: em2 {
 				compatible = "zephyr,power-state";
 				power-state-name = "suspend-to-idle";
+				exit-latency-us = <250>;
+			};
+
+			/*
+			 * EM3 seems to be exactly the same as EM2 except that
+			 * LFXO & LFRCO should be disabled, so you must use ULFRCO
+			 * as BURTC clock for the system to not lose track of time and
+			 * wake up.
+			 */
+			pstate_em3: em3 {
```

Comment:
This power state should not be here, it was removed in https://github.com/zephyrproject-rtos/zephyr/pull/94532